airbyte-integrations/bases/base-normalization/normalization/transform_catalog/stream_processor.py

```diff
@@ -268,10 +268,10 @@ def find_children_streams(self, from_table: str, column_names: Dict[str, Tuple[s
                 is_nested_array = False
                 json_column_name = column_names[field][1]
             elif is_array(properties[field]["type"]) and "items" in properties[field]:
-                quoted_field = column_names[field][1]
+                unquoted_field = column_names[field][0]
                 children_properties = find_properties_object([], field, properties[field]["items"])
                 is_nested_array = True
-                json_column_name = f"unnested_column_value({quoted_field})"
+                json_column_name = f"unnested_column_value('_{unquoted_field}')"
```

Comment:
I'm not sure about this approach, so any ideas are welcome

Reply:
You changed the naming of the nested field in the postgres macro but I don't think it's the same on the other destinations so it might break there